lab02/sections/discussion.tex

```diff
@@ -1 +1,14 @@
 \section{Discussion}\label{sec:discussion}
+A phase-lag circuit was built of an identical topology to that in Figure 1.
+The frequency of a 1V peak-to-peak input signal was varied over ~12kHz in 1kHz increments. 
+The input and output voltage and phase were measured using the in-lab oscilloscope.
+The recorded data can be seen in Table 1.
+
+In our pre-lab, we simulated the phase and amplitude response of the phase-lag system.
+As can be seen by comparing Figures XXX and XXY, the simulated results closely matched experimentally generated values.
+Both phase plots show a minimum phase of -55\si{\degree} occurring at ~3kHz. 
+The capacitors used had an error of $\pm$20$\%$.
```

Comment:
Escaping `20` from math mode isn't necessary, but it prints the same :stuck_out_tongue: